challenges/2. User Availability/README.md

```diff
@@ -41,14 +66,18 @@ const freeSlots = findCommonFreeTimes(
  * [
  *   {
  *     "start": "2017-02-21T08:00:00-05:00",
- *     "end": "2017-02-21T12:00:00-05:00"
+ *     "end": "2017-02-21T09:00:00-05:00"
+ *   },
+ *   {
+ *     "start": "2017-02-21T10:15:00-05:00",
+ *     "end": "2017-02-21T11:00:00-05:00"
  *   },
  *   {
```

Comment:
Should this be included? I read the `duration` parameter is the minimum overlapping time in minutes to include in the output.